src/python/ensembl/production/factories/metadata/genomeFactory.py

```diff
@@ -20,451 +20,536 @@
 import re
 import graphene
 from graphene_sqlalchemy import SQLAlchemyObjectType
-
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker
+from graphene_sqlalchemy.converter import convert_sqlalchemy_type
 from sqlalchemy import select
-from sqlalchemy import update
-
-from ensembl.database import DBConnection
+from sqlalchemy import JSON
 from ensembl.production.metadata.api.models.genome import Genome, GenomeDataset
 from ensembl.production.metadata.api.models.organism import Organism, OrganismGroup, OrganismGroupMember
 from ensembl.production.metadata.api.models.assembly import Assembly
 from ensembl.production.metadata.api.models.dataset import DatasetType, Dataset, DatasetSource
+from ensembl.production.metadata.api.factories.datasets import DatasetFactory
```

Comment:
No import of GenomeFactory from https://github.com/Ensembl/ensembl-metadata-api/blob/main/src/ensembl/production/metadata/api/factories/genome.py ?